CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Ability to read HGV tags from OSM Nodes (Issue #49)
 - No need to add optimisation parameter in request (PR #87)
 - Option to respond md5 of osm file used for graph generation (Issue #48)
+- Option to specify maximum locations for matrix request when using non-standard weightings (Issue #94)
```

Comment:
Should be added under [Unreleased], not under a previous release as they are not in that one